### PR TITLE
fix: golint 1.6.0 errors

### PR DESCRIPTION
**Pull request by BruceMacD**
## Summary
The lint runner is running the new golint version (1.6.0) and it is detecting some new warnings. This PR cleans up these problems.

## Checklist

- [x] Considered security implications of the change
- [x] Change is backwards compatible if it needs to be (user can upgrade without manual steps?)
- [x] Nothing sensitive logged

### CODE REVIEW

File: internal/cmd/about.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ func (t *triangle) Draw(surf *sprite.Surface) {
 		t.B.ScreenX(), t.B.ScreenY(),
 		t.C.ScreenX(), t.C.ScreenY(),
 		t.Color, true)
+	//nolint
```
Comment (BruceMacD) [inline internal/cmd/about.go:48]: This is an intentionally empty branch

File: internal/cmd/keys.go [hunk 2]
```diff
@@ -157,7 +157,7 @@ func newKeysListCmd(cli *CLI) *cobra.Command {
 				return err
 			}
 
-			keys := &api.ListResponse[api.AccessKey]{}
+			keys := &api.ListResponse[api.AccessKey]{} // nolint
```
Comment (BruceMacD) [inline internal/cmd/keys.go:160]: False positive: was being detected as unused but `keys.Items` is used

File: internal/cmd/local_server.go [hunk 3]
```diff
@@ -106,7 +109,7 @@ func formatURLMsg(msg string, fallback string) string {
 	}
 
 	formatted := strings.ReplaceAll(msg, "_", " ")
-	formatted = strings.Title(strings.ToLower(formatted))
+	formatted = cases.Title(language.English).String(formatted)
```
Comment (BruceMacD) [inline internal/cmd/local_server.go:112]: Go 1.18 deprecated `strings.Title`

**Comment by jmorganca** [comment]
Thanks for fixing this!

File: internal/cmd/about.go (hunk 1 above)
Comment (BruceMacD) [reply internal/cmd/about.go]: That worked, updated

**Review by mxyng — APPROVED** [review]

File: internal/cmd/about.go [hunk 4]
```diff
@@ -40,14 +40,12 @@ func newTriangle(a, b, c *point3D, ch rune) *triangle {
 }
 
 func (t *triangle) Draw(surf *sprite.Surface) {
-	err := surf.Triangle(
+	// nolint
+	surf.Triangle(
```
Comment (dnephin) [inline internal/cmd/about.go:44]: I think this should work to indicate "ignore error":
```suggestion
	_ = surf.Triangle(
```

File: internal/cmd/keys.go (hunk 2 above)
Comment (dnephin) [reply internal/cmd/keys.go:160]: I don't think this is a false positive. The value of `keys` is never used here, it's being overwritten.

This should change to just a variable declaration:
```suggestion
			var keys *api.ListResponse[api.AccessKey]
```

**Review by dnephin — COMMENTED** [review]
Thanks for fixing these! A couple suggestions for the fixes

File: internal/cmd/about.go (hunk 4 above)
Comment (BruceMacD) [reply internal/cmd/about.go]: This worked 👍

File: internal/cmd/about.go [hunk 5]
```diff
@@ -40,14 +40,12 @@ func newTriangle(a, b, c *point3D, ch rune) *triangle {
 }
 
 func (t *triangle) Draw(surf *sprite.Surface) {
-	err := surf.Triangle(
+	// nolint
```
Comment (dnephin) [inline internal/cmd/about.go:43]:
```suggestion
```

Do we still need this?

**Review by dnephin — APPROVED** [review]

File: internal/cmd/about.go (hunk 5 above)
Comment (dnephin) [reply internal/cmd/about.go:43]: Not a big deal, it's fine to leave it

**Pull request merged**